geode-redis/README.md

```diff
@@ -128,210 +128,61 @@ not connected>
-|                    	|                                                    	| XREVRANGE                     	|
-|                    	|                                                    	| XTRIM                         	|
-|                    	|                                                    	| ZADD                          	|
-|                    	|                                                    	| ZCARD                         	|
-|                    	|                                                    	| ZCOUNT                        	|
-|                    	|                                                    	| ZINCRBY                       	|
-|                    	|                                                    	| ZINTERSTORE                   	|
-|                    	|                                                    	| ZLEXCOUNT                     	|
-|                    	|                                                    	| ZPOPMAX                       	|
-|                    	|                                                    	| ZPOPMIN                       	|
-|                    	|                                                    	| ZRANGE                        	|
-|                    	|                                                    	| ZRANGEBYLEX                   	|
-|                    	|                                                    	| ZRANGEBYSCORE                 	|
-|                    	|                                                    	| ZRANK                         	|
-|                    	|                                                    	| ZREM                          	|
-|                    	|                                                    	| ZREMRANGEBYLEX                	|
-|                    	|                                                    	| ZREMRANGEBYRANK               	|
-|                    	|                                                    	| ZREMRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANGE                     	|
-|                    	|                                                    	| ZREVRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANK                      	|
-|                    	|                                                    	| ZSCAN                         	|
-|                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
-
-**NOTES:**
-
-[1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
-[2] - SLOWLOG is implemented as a NoOp.
+The Geode APIs compatible with Redis currently implement a subset of the full Redis command set.
+
```

Comment:
There are also a couple notes we have in the docs around supported commands. I am not sure if we want to add them here (or maybe remove them from the docs?) Specifically these two:
- `Commands not listed above are **not implemented**.` (this is at the end of the Supported Commands section)
- `**Note**: These commands are supported for Redis 5.` (at the beginning of the Supported Commands section)

Reply:
I'm not sure what we want to do about those-  I'll add them to the final check story